src/Razor/test/testapps/ComponentApp/Components/Shared/NavMenu.razor

```diff
@@ -24,6 +24,14 @@
         </li>
     </ul>
 </div>
+<SurveyPrompt Title="Foo" />
+<NavLink Match="NavLinkMatch.All" />
```

Comment:
Do we need all these?

Reply:
Good catch, part of my local  testing slipped in.